sos/report/plugins/__init__.py

```diff
@@ -1425,6 +1441,9 @@ def time_filter(path):
                 if self._is_forbidden_path(_file):
                     self._log_debug("skipping forbidden path '%s'" % _file)
                     continue
+                if self._is_skipped_path(_file):
+                    self._log_debug("skipping excluded path '%s'" % _file)
```

Comment:
Worth adding this info to `manifest.json` (or optionally to `sos.[html|txt|json` reports as well)?  (and ditto for skipped cmds)

Reply:
I don't think so, no. The manifest captures collected files, and executed commands (even if they fail). In this case we aren't executing any commands, and we aren't attempting a file collection.